guarddog/cli.py

```diff
@@ -323,98 +289,71 @@ def _list_rules(ecosystem):
     print(table)
 
 
-@cli.group
-def npm(**kwargs):
-    """Scan a npm package or verify a npm project"""
-    pass
-
-
-@cli.group
-def pypi(**kwargs):
-    """Scan a PyPI package or verify a PyPI project"""
-    pass
-
-
-@npm.command("scan")
-@common_options
-@scan_options
-@npm_options
-def scan_npm(
-    target, version, rules, exclude_rules, output_format, exit_non_zero_on_finding
-):
-    """Scan a given npm package"""
-    return _scan(
-        target,
-        version,
-        rules,
-        exclude_rules,
-        output_format,
-        exit_non_zero_on_finding,
-        ECOSYSTEM.NPM,
-    )
-
-
-@npm.command("verify")
-@common_options
-@verify_options
-@npm_options
-def verify_npm(target, rules, exclude_rules, output_format, exit_non_zero_on_finding):
-    """Verify a given npm project"""
-    return _verify(
-        target,
-        rules,
-        exclude_rules,
-        output_format,
-        exit_non_zero_on_finding,
-        ECOSYSTEM.NPM,
-    )
-
-
-@pypi.command("scan")
-@common_options
-@scan_options
-@pypi_options
-def scan_pypi(
-    target, version, rules, exclude_rules, output_format, exit_non_zero_on_finding
-):
-    """Scan a given PyPI package"""
-    return _scan(
-        target,
-        version,
-        rules,
-        exclude_rules,
-        output_format,
-        exit_non_zero_on_finding,
-        ECOSYSTEM.PYPI,
-    )
-
+class CliEcosystem(click.Group):
```

Comment:
nice one (perhaps a quick comment to explain what this is a bit more to someone who doesn't know Click)